server/routes/api/disasters.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Endpoints for disaster dashboard"""
+
+from flask import Blueprint
+import services.datacommons as dc
+import json
+import os
+import flask
+
+# Define blueprint
+bp = Blueprint("disasters", __name__, url_prefix='/disasters')
+
+EARTH_DCID = "Earth"
```

Comment:
curious what this constant is being used for?

Reply:
This file is a copy of the original disasters.py in dev. The constant was in the original file, so I didn't touch it. Let me check to see if it can be removed.

Reply:
ah, ok, so what you were looking at was the api file & what you're creating is the route file, so some things that need to be done:

- move this file to server/routes
- remove the copied code from the dev file
- nice to have: rename the bp in disaster_dashboard.py to "disaster-api" and update the naming where the bp is registered in the __init__.py

Reply:
Update: removed! Thanks for the catch!

Reply:
Further update: Made the changes!

* disasters.py is now in server/routes
* no more unused constant
* Renamed disaster_dashboard bp to `disaster_api`
* Updated the registry in __init__